web-console/knikubevirt/vm-templates/vm-templates.md

```diff
@@ -0,0 +1,86 @@
+# Virtual machine templates
+
+### Summary
+
+- VM-templates is located at the main side menu, in 'Workloads' under 'virtual machines'.
```

Comment:
for consistency can we have this say "Virtual Machine Templates" are located under "Virtual Machines" in the primary navigation.